Content.Server/Entry/EntryPoint.cs

```diff
@@ -47,6 +47,7 @@ public sealed class EntryPoint : GameServer
         private PlayTimeTrackingManager? _playTimeTracking;
         private IEntitySystemManager? _sysMan;
         private IServerDbManager? _dbManager;
+        private ConnectionManager? _connectionManager;
```

Comment:
You should make `void Update()` part of the `IConnectionManager` interface, not resolve the concrete implementation here.